day-04/exercise_set_4.ipynb

```diff
@@ -426,31 +363,19 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 16,
+   "execution_count": null,
    "metadata": {
     "id": "7kkxk3lCAreD"
    },
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "array([[1, 4],\n",
-       "       [5, 7]])"
-      ]
-     },
-     "execution_count": 16,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "outputs": [],
    "source": [
     "A = np.array([[1, 4], [5, 7]])\n",
     "A"
```

Comment:
```suggestion
    "print(A)"
```

Reply:
I guess this is OK, as we have mentioned this functionality in exercise 1 : )

Reply:
It just looks weird without outputs, but I agree that it should be fine.